src/Client/Web/Bit.Client.Web.BlazorUI/Components/Links/BitLink.razor.cs

```diff
@@ -14,7 +15,7 @@ public partial class BitLink
 
         [Parameter] public EventCallback<MouseEventArgs> OnClick { get; set; }
 
-        protected override string RootElementClass => "bit-link";
+        protected override string RootElementClass => Href.HasValue() || !IsEnabled ? "bit-lnk"  : "bit-lnk-btn";
```

Comment:
what is this change for?

Reply:
an issue is made for this change 
https://github.com/bitfoundation/bitframework/issues/310